docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -43,18 +43,19 @@ <h2>Release notes: Open GEE 5.2.5</h2>
   <h5>
```

Comment:
let's remove "Open GEE 5.2.5 is currently in development"

Reply:
Removed.